<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cajeto ATM Bancolombia</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn20">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>$ 20.000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn50">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>$ 50.000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn100">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>$ 100.000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn200">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>$ 200.000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn300">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>$ 300.000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn500">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>$ 500.000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1000">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>$ 1.000.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tittle2_frame1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Selecciona el monto a tu </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tittle_frame1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Transacci√≥n</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_bancolombia_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-30</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/bancolombialogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_image_billete_50">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>321</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/50.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>430</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_billete_100">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>420</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>239</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_billete_20">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>230</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_billete_50">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>330</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image_billete_20">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/20.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image_billete_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/10.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_billete_10">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>135</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>340</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x1">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>141</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image_billete_100">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/100.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_volver_a_retirar">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>510</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Volver a retirar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>29</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Porfavor retire su dinero</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_solicitar_monto_diferente">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Solicitar un valor diferente</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>751</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="txt_valor_diferente">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Ej: 40000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_aceptar">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>490</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_tittle_frame2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ingrese monto a retirar</string>
     </property>
    </widget>
   </widget>
   <zorder>btn_solicitar_monto_diferente</zorder>
   <zorder>frame_2</zorder>
   <zorder>btn20</zorder>
   <zorder>btn50</zorder>
   <zorder>btn100</zorder>
   <zorder>btn200</zorder>
   <zorder>btn300</zorder>
   <zorder>btn500</zorder>
   <zorder>btn1000</zorder>
   <zorder>lbl_tittle2_frame1</zorder>
   <zorder>lbl_tittle_frame1</zorder>
   <zorder>lbl_bancolombia_logo</zorder>
   <zorder>frame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
